lms/static/scripts/frontend_apps/components/dashboard/DashboardApp.tsx

```diff
@@ -1,11 +1,16 @@
 import classnames from 'classnames';
-import { Route, Switch } from 'wouter-preact';
+import { Route, Switch, useParams } from 'wouter-preact';
 
 import AssignmentActivity from './AssignmentActivity';
 import CourseActivity from './CourseActivity';
 import DashboardFooter from './DashboardFooter';
+import OrganizationActivity from './OrganizationActivity';
 
 export default function DashboardApp() {
+  const { organizationPublicId } = useParams<{
+    organizationPublicId: string;
+  }>();
```

Comment:
`useParams()` does not return matching params from parent routes when used inside a nested route, so we need to call this here to get the organization ID, and then pass it down as regular props to `<OrganizationActivity />`.

Reply:
Maybe the parent route should just be `/dashboard`, and we should add the `/organizations/:organizationPublicId` in all routes here.